src/server/pachyderm_test.go

```diff
@@ -689,7 +689,7 @@ func TestPipelineThatOverwritesFile(t *testing.T) {
 	require.Equal(t, 1, len(outCommits))
 	var buffer2 bytes.Buffer
 	require.NoError(t, c.GetFile(outRepo.Name, outCommits[0].Commit.ID, "file", 0, 0, "", nil, &buffer2))
-	require.Equal(t, "foo\nfoo\nfoo\nfoo\nfoo\nfoo\n", buffer2.String())
+	require.Equal(t, "foo\nfoo\nfoo\n", buffer2.String())
```

Comment:
That makes sense. I'm all for rewriting tests based on intent.

Those PutFiles threw me off. Reading the test closer, it looks like `cp` command causes the change in the output, which suggests its making a call to `SetAttr`. That makes sense. If I cp a file someplace I'm overwriting it, which we now support.

Thanks for the detour on behalf of me learning :-)